toolkit/tools/imagegen/configuration/configuration.go

```diff
@@ -66,6 +66,9 @@ type Config struct {
 	Disks         []Disk         `json:"Disks"`
 	SystemConfigs []SystemConfig `json:"SystemConfigs"`
 
+	// Shorthand for when there is only a single SystemConfig value.
+	SystemConfig *SystemConfig `json:"SystemConfig"`
```

Comment:
We have a `DefaultSystemConfig *SystemConfig` in our existing package (although its a bit of a hack, used in the ISO installer mostly).

Just an FYI so there isn't any confusion between the two.